common/lib/xmodule/xmodule/modulestore/tests/django_utils.py

```diff
@@ -198,6 +198,49 @@ def xml_store_config(data_dir, source_dirs=None):
 )
 
 
+class ModuleStoreClassCleanupTestCase(TestCase):
+    """
+    Sometimes you want to create a course and know it will stick around between
+    setUp()s.
+
+    Note that if you create any Django ORM data in your setUpClass() methods,
+    you must manually clean it up in your tearDownClass(). Django's built-in
+    TestCase class (which this class inherits from) will cleanup the database
+    between tests on databases that support transactions, but it cannot clean
+    up models created in setUpClass(). In Django 1.8, we will be able to use
+    setUpTestData() for this.
```

Comment:
I'd probably just have this documentation recommend using `setUp` for DjangoORM, rather than `setUpClass`, since the ORM is all done within a transaction and should be super fast.